css/cmn.css

```diff
@@ -0,0 +1,184 @@
+html{
+  background-color: #191919;
+}
+@font-face{
+	  font-family: 'Multicolore';
+	  src: url(/fonts/multicolore/Multicolore.otf);
+  }
+@font-face{
+  font-family: 'Biko';
+  src: url('/fonts/biko/Biko_Regular.otf');
+}
```

Comment:
Could this be moved into a file that both PHP files reference?  That way, it will be easier to modify the styles for the app as a whole.